src/components/Sidebar/NavContent.jsx

```diff
@@ -93,6 +97,21 @@ function NavContent() {
                 </Typography>
               </Link>
 
+              <Link
+                component={NavLink}
+                id="zap-nav"
+                to="/zap"
+                isActive={(match, location) => {
+                  return checkPage(match, location, "zap");
+                }}
+                className={`button-dapp-menu ${isActive ? "active" : ""}`}
+              >
+                <Typography variant="h6">
+                  <SvgIcon component={ZapIcon} color="primary" />
+                  OlyZaps
```

Comment:
add `<Trans>`

Reply:
Why does the name need Trans?

Reply:
ahhaah I guess you're right. but the other strings on the page should